package.json

```diff
@@ -4,6 +4,8 @@
   "scripts": {
     "server": "ts-node server/server.js",
     "server:debug": "node --inspect -r ts-node/register ./server/server.js",
+    "widget": "cd ./ui && npm run dev",
+    "combined": "npx --yes npm-run-all --parallel server widget",
```

Comment:
My only thought is that using "combined" as the command means you need to really understand what's happening behind the scenes. What about `ucw` or `ucw-app`? If I were a developer who didn't read the docs, and I saw these options, it may be more intuitive than "combined". Just a thought, I could be wrong 🤷

Reply:
Good call-out. I'll change it.